src/docfx/build/context/Output.cs

```diff
@@ -95,6 +95,15 @@ public void Delete(string destRelativePath)
             {
                 File.Delete(destinationPath);
             }
+
+            if (legacy)
```

Comment:
it's only works for legacy?  what about ideal output?

Reply:
For ideal, we don't have `.mta.json` so there is no problem. There is also an existing tests that covers conflicts for ideal output